### ISSUE TITLE
Simple messaging

### ISSUE THREAD

**Issue by danwdart** [dandart/projectchaplin#60]
Message button on user page
Messages with unread count in top right
List of messages and ability to reply page

**Comment by dandart** [comment]
Do we  need this? It's a bit out of scope methinks.

**Comment by danwdart** [comment]
I agree with 2.0 - 2.0 being "out of scope right now" things, like icebox
